interfaces.go

```diff
@@ -40,12 +40,13 @@ type Growth interface {
 	GetFreeTrialInfo(context.Context, int32) ([]*GrowthPromoCodeClaim, error)
 }
 
-// User service allows managing users in Confluent Cloud
-type UserInterface interface {
-	List(context.Context) ([]*User, error)
-	GetServiceAccounts(context.Context) ([]*User, error)
-	GetServiceAccount(context.Context, int32) (*User, error)
-	LoginRealm(context.Context, *GetLoginRealmRequest) (*GetLoginRealmReply, error)
+// Schema Registry service allows managing SR clusters in Confluent Cloud
+type SchemaRegistry interface {
+	CreateSchemaRegistryCluster(context.Context, *SchemaRegistryClusterConfig) (*SchemaRegistryCluster, error)
```

Comment:
Not a huge issue, but this is a bit redundant.
```suggestion
	CreateCluster(context.Context, *SchemaRegistryClusterConfig) (*SchemaRegistryCluster, error)
```